### PR TITLE
MAINT: fix gg2 URLs & add human stool weighted classifiers

### PR DESCRIPTION
**Pull request by lizgehret**
- fixes URLs for gg2 links that were moved from classifiers -> release/common
- adds URLs for human stool weighted silva & gtdb

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -45,6 +45,18 @@ Taxonomic classifiers perform best when they are trained based on your specific
 **Citations**: @Robeson2020-ax, @Bokulich2018-yb, [Silva](#Silva-Citations)
 ```
 
+```{card}
+:header: **Silva 138 99% OTUs full-length weighted sequences**
```
Comment (cherman2) [inline index.md]:
```suggestion
:header: **human stool weighted Silva 138 99% OTUs full-length sequences**
```

File: index.md [hunk 2]
```diff
@@ -56,10 +68,20 @@ Taxonomic classifiers perform best when they are trained based on your specific
 **Citations**: @Parks2021, @Parks2020, @Parks2018, @Rinke2021
 ```
 
+```{card}
+:header: **GTDB r220 full-length weighted sequences**
```
Comment (cherman2) [inline index.md]:
```suggestion
:header: **human stool weighted GTDB r220 full-length sequences**
```

File: index.md [hunk 3]
```diff
@@ -45,6 +45,18 @@ Taxonomic classifiers perform best when they are trained based on your specific
 **Citations**: @Robeson2020-ax, @Bokulich2018-yb, [Silva](#Silva-Citations)
 ```
 
+```{card}
+:header: **Silva 138 99% OTUs full-length weighted sequences**
+
+**Download**: [Silva 138 99% OTUs full-length weighted sequences](https://data.qiime2.org/2024.5/common/silva-138-99-nb-weighted-classifier.qza)\
```
Comment (cherman2) [inline index.md]:
```suggestion
**Download**: [human stool weighted Silva 138 99% OTUs full-length sequences](https://data.qiime2.org/2024.5/common/silva-138-99-nb-weighted-classifier.qza)\
```

File: index.md [hunk 4]
```diff
@@ -56,10 +68,20 @@ Taxonomic classifiers perform best when they are trained based on your specific
 **Citations**: @Parks2021, @Parks2020, @Parks2018, @Rinke2021
 ```
 
+```{card}
+:header: **GTDB r220 full-length weighted sequences**
+**Download**: [GTDB r220 full-length weighted sequences](https://data.qiime2.org/2024.5/common/gtdb_weighted_classifier_r220.qza)\
```
Comment (cherman2) [inline index.md]:
```suggestion
**Download**: [human stool weighted GTDB r220 full-length sequences](https://data.qiime2.org/2024.5/common/gtdb_weighted_classifier_r220.qza)\
```

**Review by cherman2 — APPROVED** [review]

**Comment by lizgehret** [comment]
Thanks @cherman2! We can get the additional weighted classifiers retrained & added next week - but I'll get this merged in the meantime, and update the AWS/data302 URLs to include human stool in their filenames.

File: index.md [hunk 5]
```diff
@@ -45,6 +45,18 @@ Taxonomic classifiers perform best when they are trained based on your specific
 **Citations**: @Robeson2020-ax, @Bokulich2018-yb, [Silva](#Silva-Citations)
 ```
 
+```{card}
+:header: **human stool weighted Silva 138 99% OTUs full-length sequences**
+
+**Download**: [human stool weighted Silva 138 99% OTUs full-length sequences](https://data.qiime2.org/2024.5/common/silva-138-99-nb-weighted-classifier.qza)\
```
Comment (lizgehret) [inline index.md]:
```suggestion
**Download**: [human stool weighted Silva 138 99% OTUs full-length sequences](https://data.qiime2.org/2024.5/common/silva-138-99-nb-human-stool-weighted-classifier.qza)\
```

File: index.md [hunk 6]
```diff
@@ -56,10 +68,20 @@ Taxonomic classifiers perform best when they are trained based on your specific
 **Citations**: @Parks2021, @Parks2020, @Parks2018, @Rinke2021
 ```
 
+```{card}
+:header: **human stool weighted GTDB r220 full-length sequences**
+**Download**: [human stool weighted GTDB r220 full-length sequences](https://data.qiime2.org/2024.5/common/gtdb_weighted_classifier_r220.qza)\
```
Comment (lizgehret) [inline index.md]:
```suggestion
**Download**: [human stool weighted GTDB r220 full-length sequences](https://data.qiime2.org/2024.5/common/gtdb_human_stool_weighted_classifier_r220.qza)\
```

**Comment by lizgehret** [comment]
associated data302 PR: https://github.com/qiime2/data302/pull/151

**Pull request merged**